src/gui/dialogs/editor/custom_tod.cpp

```diff
@@ -129,15 +129,15 @@ void custom_tod::pre_show(window& window)
 
 	connect_signal_notify_modified(
 			*(color_field_r_->get_widget()),
-			std::bind(&custom_tod::color_slider_callback, this));
+			std::bind(&custom_tod::color_slider_r_callback, this));
 
 	connect_signal_notify_modified(
 			*(color_field_g_->get_widget()),
-			std::bind(&custom_tod::color_slider_callback, this));
+			std::bind(&custom_tod::color_slider_g_callback, this));
 
 	connect_signal_notify_modified(
 			*(color_field_b_->get_widget()),
-			std::bind(&custom_tod::color_slider_callback, this));
+			std::bind(&custom_tod::color_slider_b_callback, this));
```

Comment:
I get the feeling that these could all use the same callback, but with different bound parameters. Does that seem worthwhile? I guess it would require the use of a pointer to member, so it's not clear whether it's an improvement…

Reply:
Maybe just add one more parameter to `color_slider_callback`? Like `std::bind(&custom_tod::color_slider_callback, this, "r")`? I'll test it. But it might fail if `connect_signal_notify_modified()` cannot attach the same function to multiple widgets.

Reply:
That wasn't precisely what I had in mind, but it's similar enough.

Reply:
Merged to one callback with parameter. Please see latest commit.